variables.tf

```diff
@@ -481,3 +481,268 @@ variable "graphdb_enable_userdata_scripts_on_reboot" {
+}
+
+variable "s3_cmk_alias" {
+  description = "The alias for the CMK key."
+  type        = string
+  default     = "alias/graphdb-s3-cmk-key"
+}
+
+variable "s3_kms_key_enabled" {
+  description = "Specifies whether the key is enabled."
+  type        = bool
+  default     = true
+}
+
+variable "s3_key_specification" {
+  description = "Specification of the Key."
+  type        = string
+  default     = "SYMMETRIC_DEFAULT"
+}
+
+variable "s3_key_deletion_window_in_days" {
+  description = "The waiting period, specified in number of days for AWS to delete the KMS key(Between 7 and 30)."
+  type        = number
+  default     = 30
+}
+
+variable "s3_cmk_description" {
+  description = "Description for the KMS Key"
+  type        = string
+  default     = "KMS key for S3 bucket encryption."
+}
+
+variable "s3_external_kms_key_arn" {
+  description = "Externally provided KMS CMK"
+  type        = string
+  default     = ""
+}
+
+# Parameter store KMS encryption
+
+variable "graphdb_parameter_store_cmk_alias" {
+  description = "The alias for the CMK key."
+  type        = string
+  default     = "alias/graphdb-param-cmk-key"
+}
+
+variable "graphdb_parameter_store_key_admin_arn" {
+  description = "ARN of the key administrator role for Parameter Store"
+  type        = string
+  default     = ""
+}
+
+variable "graphdb_parameter_store_key_tags" {
+  description = "A map of tags to assign to the resources."
+  type        = map(string)
+  default     = {}
+}
+
+variable "graphdb_parameter_store_key_rotation_enabled" {
+  description = "Specifies whether key rotation is enabled."
+  type        = bool
+  default     = true
+}
+
+variable "graphdb_parameter_store_default_key" {
+  description = "Define default key for parameter store if no KMS key is used"
+  type        = string
+  default     = "alias/aws/ssm"
+}
+
+variable "graphdb_parameter_store_key_enabled" {
+  description = "Specifies whether the key is enabled."
+  type        = bool
+  default     = true
+}
+
+variable "graphdb_parameter_store_key_spec" {
+  description = "Specification of the Key."
+  type        = string
+  default     = "SYMMETRIC_DEFAULT"
+}
+
+variable "graphdb_parameter_store_key_deletion_window_in_days" {
+  description = "The waiting period, specified in number of days for AWS to delete the KMS key(Between 7 and 30)."
+  type        = number
+  default     = 30
+}
+
+variable "graphdb_parameter_store_cmk_description" {
+  description = "Description for the KMS Key"
+  type        = string
+  default     = "KMS key for Parameter Store bucket encryption."
+}
+
+variable "create_graphdb_parameter_store_kms_key" {
+  description = "Enable creation of KMS key for Parameter Store encryption"
+  type        = bool
+  default     = false
+}
+
+variable "graphdb_parameter_store_external_kms_key" {
+  description = "Externally provided KMS CMK"
+  type        = string
+  default     = ""
+}
+
+# GraphDB EBS KMS encryption
+
+variable "graphdb_ebs_key_admin_arn" {
```

Comment:
We already have `ebs_` variables, do not prefix with `graphdb_`. Or update all other variables (breaking change).